src/temaWeek7Comparator/Persoane.java

```diff
@@ -3,7 +3,6 @@
  * Implemented 2 comparators on a treeSet of persons:NameComparator and AgeComparator
  */
 
-
 import java.util.*;
 
 public class Persoane {
```

Comment:
Same remark as above; different class for main().
Also, please remember to use the built in code formatter.